thorlcr/thorutil/thormisc.cpp

```diff
@@ -1337,5 +1363,40 @@ IPerfMonHook *createThorMemStatsPerfMonHook(CJobBase &job, int maxLevel, IPerfMo
     return new CPerfMonHook(job, maxLevel, chain);
 }
 
-
 const StatisticsMapping spillStatistics(StTimeSpillElapsed, StTimeSortElapsed, StNumSpills, StSizeSpillFile, StKindNone);
+
+bool isOOMException(IException *_e)
+{
+    if (_e)
+    {
+        IThorException *e = QUERYINTERFACE(_e, IThorException);
+        IException *oe = e && e->queryOriginalException() ? e->queryOriginalException() : _e;
+        int ecode = oe->errorCode();
+        if (ecode >= ROXIEMM_ERROR_START && ecode <= ROXIEMM_ERROR_START) // should have ROXIEMM_ERROR_END marker probably
```

Comment:
This range doesn't look correct.  Should probably add a ROXIEMM_ERROR_START to errorlist.h